src/django/api/templates/mail/claim_facility_submitted_body.txt

```diff
@@ -1,9 +1,16 @@
 {% block content %}
 Hi,
 
-You're receiving this email because we received your request to claim an Open Supply Hub facility. We'll review the request and be in touch soon.
+You're receiving this email because you were listed as the contact for a facility claim request on Open Supply Hub. We'll review the request and be in touch soon. The expected processing time for the initial review of a claim is 10 - 14 days. 
 
-Here is the information submitted with your request:
+Vendors can refer facilities to claim their profiles, however, the Facility Claim request will only be approved when it is submitted by a facility owner and/or senior management staff at the facility location in question.  If this claim is a referral, please visit our website [https://opensupplyhub.org/auth/register], and register as a user on Open Supply Hub. 
+
+To complete a claim request, you'll need to submit the following documentation:
```

Comment:
Looks like the same text should be added to the HTML template (`claim_facility_submitted_body.html`), doesn't it?
Sorry, if I missed smth.